R/check-input-helpers.R

```diff
@@ -166,7 +166,7 @@ check_columns_present <- function(data, columns) {
 #' @keywords internal_input_check
 test_columns_present <- function(data, columns) {
   check <- check_columns_present(data, columns)
-  return(is.logical(check))
+  return(isTRUE(check))
```

Comment:
Mmhhh interesting! Why was `is.logical()` use here? it returns TRUE for both `TRUE` and `FALSE` whereas `isTRUE` will only returns `TRUE` for a value of `TRUE`.